h/data_tasks/report/create_from_scratch/08_annotation_counts/01_create_view.sql

```diff
@@ -0,0 +1,55 @@
+-- This is a summary table which re-interprets the annotations data in terms of
+-- weekly counts split by a number of facets:
+--
+--  * Authority
+--  * Group
+--  * User
+--  * Annotation sub-type
+--  * Annotation shared status
+--
+-- This is intended to speed up a variety of summary queries over the table and
+-- make queries about the sub-type easy.
+
+DROP MATERIALIZED VIEW IF EXISTS report.annotation_counts CASCADE;
```

Comment:
Ok, so the `create_view` script starts by dropping the `annotation_counts` table in case the same `create_views` script has already been run before so the table already exists

Reply:
Yeah. These scripts are only run during the recreate type actions. For repeatability it's best if they destroy everything they create.

It's not _always_ required in a full run, as one of the first things we often do is drop the schema. In the case of `h` though, we have `fast_recreate` which leaves the `annotations` view and then redefines everything else.